libpod/networking_linux.go

```diff
@@ -249,9 +249,33 @@ func (r *Runtime) teardownNetNS(ctr *Container) error {
 	return nil
 }
 
+func getContainerNetNS(ctr *Container) (string, error) {
+	if ctr.state.NetNS != nil {
+		return ctr.state.NetNS.Path(), nil
+	}
+	if ctr.config.NetNsCtr != "" {
+		c, err := ctr.runtime.LookupContainer(ctr.config.NetNsCtr)
```

Comment:
NetNSCtr is required to be the full ID, to set can use GetContainer() here, which should be a bit faster faster